compiler/src/dotty/tools/backend/jvm/DottyBackendInterface.scala

```diff
@@ -306,6 +306,11 @@ class DottyBackendInterface(outputDirectory: AbstractFile, val superCallsMap: Ma
         val desc = innerClasesStore.typeDescriptor(typ.asInstanceOf[bcodeStore.int.Type]) // the class descriptor of the nested annotation class
         val nestedVisitor = av.visitAnnotation(name, desc)
         emitAssocs(nestedVisitor, assocs, bcodeStore)(innerClasesStore)
+
```

Comment:
I think we can avoid the duplication by checking against tpd.RefTree instead of tpd.Select above. Can you try that?

Reply:
Indeed, it worked